docs/source/apis/saas_api.rst

```diff
@@ -0,0 +1,151 @@
+   - **Email**: Provide a valid email address where we can send you confirmations and communications.
+   - **Company**: Enter the name of your company or organization.
+
+3. Optional Preferences:
+
+   - [] Check the box to sign up for weekly usage reports.
+   - [] Check the box to sign up for future marketing emails.
+
+4. Terms and Conditions:
+
+   - [] Read and accept our terms and conditions by checking the appropriate box to proceed.
+
+5. Once all information is entered and preferences and terms are accepted, click the "Sign Up" button to create your account.
+
+.. image:: imgs/paid_api/sign_up_placeholder.png
+  :align: center
+  :alt: Sign Up Form
+
+
+Email Confirmation and Payment Processing
+-----------------------------------------
+
+6. Upon successful sign-up, you will receive an email from Stripe.
+
+   - There will be an initiation fee that will be collected upfront for all new users. Please refer to the pricing information page on our website.
+7. Click the "Pay this invoice" button in the email and follow the instructions to enter your payment details securely on Stripe website.
+8. Confirm the payment to complete this process. Look out for any confirmation prompts or messages that signify a successful transaction.
+
+   - Note that we don't send out a receipt. If you want to keep a receipt, click the "Download receipt" button after completing the payment on Stripe.
+
+
+Using the API Key and URL
+--------------------------
+
+1. After your payment is processed, you will receive an email from us containing your unique API credentials.
+2. Check your inbox (and the spam folder, just in case) for an email from our domain.
+3. Save these credentials securely, as you will need them to access and integrate our API with your applications.
+
+
+Update Billing
+--------------
+
+If you need to update your billing information, fill out the "Update Billing" form.
+
+1. Select "Update Billing" on `Unstructured SaaS API <https://unstructured.io/api-key-hosted>`_ webpage
+2. Upon successful sign-up, you will be redirected to the Stripe payment page.
+3. Follow the instructions to enter your payment details securely.
+4. Confirm the payment to complete this process. Look out for any confirmation prompts or messages that signify a successful transaction.
+
+.. image:: imgs/paid_api/update_billing.png
+  :align: center
+  :alt: Update Billing
+
+
+Opt-in to Marketing and Usage Emails
+------------------------------------
+
+To receive regular updates on usage statistics and marketing emails, please follow these steps:
+
+1. Select "Subscribe to Usage Updates" on `Unstructured SaaS API <https://unstructured.io/api-key-hosted>`_ webpage.
+2. Enter your email address in the provided field.
+3. Click the "Submit" button to complete the subscription process.
+
+You will receive a confirmation email shortly after submitting your information. Please click the confirmation link to complete your opt-in to usage and marketing emails.
```

Comment:
@treighton and @amanda103 Could you confirm if the user will receive a confirmation email after submitting the opt-in form?